src/services/requestService.js

```diff
@@ -89,6 +89,35 @@ class RequestService {
       throw error;
     }
   }
+
+  /**
+   * Get requests by user
+   * @param {string} requestId to be rejected
+   * @param {string} status rejected or accepted
+   * @return {object} Oject of request if found
+   */
+  static async rejectUpdateRequest(requestId, status) {
```

Comment:
what if we update the name of this function to 
`updateRequest` as it's just updating a request